tools/liveimage-mount

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/python -tt
+# coding: latin-1
+# 2015-07-23 13:03:05
 #
-# liveimage-mount: Mount a LiveOS at the specified point, and log
-# into a shell.
+# liveimage-mount: Mount a LiveOS at the specified point, and log into a shell.
 #
-# Copyright 2011, Red Hat  Inc.
+# Copyright 2011, 2015 Red Hat, Inc. & Sugar Labs®
```

Comment:
I'm not sure about adding Sugar Labs here. I'm not a license expert so maybe we should ask Spot?